### DISCUSSION TITLE
I want to do some things for Deno.

### DISCUSSION THREAD

**Discussion by iam-frankqiu** [denoland/deno#8816] [Ideas]
It seems like there isn't an official repository about Deno resources like `awesome-deno`. I want to collect some good resources and packages with Deno to make people know more about Deno. I think it will make Deno more popular and more convenient for developers to know about the Deno community.  So I want to join Denoland to do it. It will be my pleasure if I can join the Denoland org to do these things. Thank you.
@ry @bartlomieju @lucacasonato @piscisaureus

**Comment by ry** [comment]
There's an "awesome-deno" at https://github.com/denolib/awesome-deno 
Contributions are welcome.

**Reply by iam-frankqiu** [reply]
> There's an "awesome-deno" at https://github.com/denolib/awesome-deno
> Contributions are welcome.

Thank you. But this repository is forked from another repository. It's hard for users to search on GitHub. Because we usually don't search a forked repository. Maybe we should create another `awesome-deno` for us to easily search. Maybe we could in this Denoland org to create an official `awesome-deno`. what do you think?

**Reply by kitsonk** [reply]
It is better left to the community at large.

**Reply by mjrlowe** [reply]
FYI @iam-frankqiu, the `awesome-deno` repository [has now been detached](https://github.com/denolib/awesome-deno/issues/190), and shows up in search results.
(Other than the official denoland repositories, it's the first repository that comes up when you search "deno" now.)

**Reply by iam-frankqiu** [reply]
> FYI @iam-frankqiu, the `awesome-deno` repository [has now been detached](https://github.com/denolib/awesome-deno/issues/190), and shows up in search results.
> (Other than the official denoland repositories, it's the first repository that comes up when you search "deno" now.)

Thank you. I have starred the repository.
